GitUI/BranchTreePanel/RepoObjectsTree.ContextActions.cs

```diff
@@ -143,6 +144,16 @@ private void ContextMenuRemoteSpecific(ContextMenuStrip contextMenu)
                 return;
             }
 
+            _remoteBranchMenuItems.ForEach(i => i.Item.Visible = !areMultipleBranchesSelected);
+
+            // TODO toggle Visible instead of Enabled for consistency with other menu items?
```

Comment:
I suggest this comment is removed

Reply:
@gerhardol I kept that comment there because the current `Enabled`-toggling of menu items following this comment is not consistent with the rest of them, for which `Visible` is toggled instead. 

My open question concerning this from our code discussion above:
> I understand your preference is to disable unusable context menu items rather than hide them. Should I apply this pattern to all menu items then? Please note that those for local branches were hidden instead of disabled before.

Now I understand you've moved away from your preference. Is that correct? Then I'd toggle `Visible` on those menu items instead as well as remove the comment.

Reply:
Personally, I have not changed but the implemented behavior follows the pattern in this component, so I am fine with it.
Fetch and Checkout related should be handled the same.
(I believe there were something additional initially, that I have forgotten about now and that we can ignore.)
So change to Visible